packages/tx/src/eip2930Transaction.ts

```diff
@@ -0,0 +1,385 @@
+import {
+  Address,
+  BN,
+  bnToHex,
+  bnToRlp,
+  bufferToHex,
+  ecrecover,
+  keccak256,
+  rlp,
+  setLengthLeft,
+  toBuffer,
+} from 'ethereumjs-util'
+import { BaseTransaction } from './baseTransaction'
+import {
+  AccessList,
+  AccessListBuffer,
+  AccessListItem,
+  isAccessList,
+  JsonTx,
+  TxData,
+  TxOptions,
+} from './types'
+
+// secp256k1n/2
+const N_DIV_2 = new BN('7fffffffffffffffffffffffffffffff5d576e7357a4501ddfe92f46681b20a0', 16)
+
+type EIP2930ValuesArray = [
+  Buffer,
+  Buffer,
+  Buffer,
+  Buffer,
+  Buffer,
+  Buffer,
+  Buffer,
+  AccessListBuffer,
+  Buffer?,
+  Buffer?,
+  Buffer?
+]
+
+export default class EIP2930Transaction extends BaseTransaction<EIP2930Transaction> {
+  public readonly chainId: BN
+  public readonly accessList: AccessListBuffer
+  public readonly v?: BN
+  public readonly r?: BN
+  public readonly s?: BN
+
+  get transactionType(): number {
+    return 1
+  }
+
+  public readonly AccessListJSON: AccessList
+
+  // EIP-2930 alias for `s`
+  get senderS() {
+    return this.s
+  }
+
+  // EIP-2930 alias for `r`
+  get senderR() {
+    return this.r
+  }
+
+  // EIP-2930 alias for `v`
+
+  get yParity() {
+    return this.v
+  }
```

Comment:
Not sure, are these new aliases really worth to be introduced or do they not just complicate working with signatures since it confuses on generic usage if someone gets used to use e.g. `yParity` instead of `v`. I would have a tendency to just remove (or - not sure - add everything to `BaseTransaction` and at some point make `v`, `r`, `s` private further down the road and therefore have theses aliases just for all tx types).

Reply:
I added this to be consistent with the EIP, it is basically an utility method.